ui/internal/tty/tty.go

```
@@ -0,0 +1,60 @@
+
+var firstPaintOnce sync.Once
+
+// FirstPaint writes the bench hook line exactly once, only under RT_UI_BENCH=1.
+func FirstPaint() { firstPaintTo(os.Stderr) }
+
+func firstPaintTo(w io.Writer) {
+	if os.Getenv("RT_UI_BENCH") != "1" {
+		return
+	}
+	firstPaintOnce.Do(func() {
+		fmt.Fprintf(w, "first-paint %d\n", time.Now().UnixMilli())
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)